hedera-mirror-rest/config/application.yml

```
@@ -87,7 +87,8 @@ hedera:
       response:
         compression: true
         headers:
-          default: { "cache-control": "public, max-age=1" }
+          default: { "cache-control": "public, max-age=1",
+          "Content-type": "application/json; charset=utf-8"}
```

Use consistent casing.
```suggestion
          "content-type": "application/json; charset=utf-8"}
```

---

Actually other places in the code so you `"Content-type"` but will change it here.